src/core/thread/mesh_forwarder.cpp

```diff
@@ -1741,7 +1741,7 @@ const char *MeshForwarder::MessageActionToString(MessageAction aAction, Error aE
     static const char *const kMessageActionStrings[] = {
         "Received",                    // (0) kMessageReceive
         "Sent",                        // (1) kMessageTransmit
-        "Prepping indir tx",           // (2) kMessagePrepareIndirect
+        "Preparing indir tx",          // (2) kMessagePrepareIndirect
```

Comment:
Changing this  is ok.  

But I don't think it was typo actually. We used this to keep it slightly shorter (to fit some max log line and not clip stuff at the end).  "Prepping" seems to be valid word 😄 

> prepping
> the action or process of preparing something, or preparing for something.
> "after hours of prepping and cooking and stressing and a few near-breakdowns, Thanksgiving was wonderful"

Reply:
Thanks! That's new knowledge for me. Then I think we don't need this change.